db/migrate/20150520144034_add_coupon_to_member.rb

```diff
@@ -0,0 +1,5 @@
+class AddCouponToMember < ActiveRecord::Migration
+  def change
+    add_column :members, :coupon, :string
```

Comment:
Spacing is just rails terrible idea that lining things up is important. It was the column limits I was confused about but that might be sqlite.

Pretty sure the column limits don't equate to decimal digits directly.